components/article/test/templates.coffee

```diff
@@ -207,10 +207,10 @@ describe 'article show template', ->
                   name: "October Gallery",
                   slug: "october-gallery"
                 },
-                artist: {
+                artists: [{
                   name: "Govinda Sah 'Azad'",
                   slug: "govinda-sah-azad"
-                }
+                }]
               }
             ]
           }
```

Comment:
Nice, could we add an assertion to this test to make sure multiple `artists` get rendered correctly?